ui/v2.5/src/locales/en-GB.json

```diff
@@ -281,7 +281,7 @@
       "check_for_insecure_certificates_desc": "Some sites use insecure ssl certificates. When unticked the scraper skips the insecure certificates check and allows scraping of those sites. If you get a certificate error when scraping untick this.",
       "chrome_cdp_path": "Chrome CDP path",
       "chrome_cdp_path_desc": "File path to the Chrome executable, or a remote address (starting with http:// or https://, for example http://localhost:9222/json/version) to a Chrome instance.",
-      "create_galleries_from_folders_desc": "If true, creates galleries from folders containing images.",
+      "create_galleries_from_folders_desc": "If true, creates galleries from folders containing images by default. Create a File called .forceGallery or .noGallery in a folder to enforce/prevent this.",
```

Comment:
The in-app help section should also be updated with a paragraph explaining why someone would want to use this new feature.

Reply:
Valid Point. Added a paragraph explaining the functionality, based on the updated codebase.